pkg/velaql/providers/query/handler.go

```diff
@@ -107,6 +107,29 @@ func (h *provider) ListResourcesInApp(ctx wfContext.Context, v *value.Value, act
 	return v.FillObject(appResList, "list")
 }
 
+// ListAppliedResources list applied resource from tracker
+func (h *provider) ListAppliedResources(ctx wfContext.Context, v *value.Value, act types.Action) error {
```

Comment:
I think we can use ListApplicationResources. Since the `appliedResources` usually refers to the .Status.appliedResource in application, the query here is finding resources in ResourceTracker, it is a bit confusing to use `ListAppliedResources` here.

Reply:
This function's purpose is to replace `.Status. appliedResource`. so, I used the same name.

Reply:
But we still need .Status.appliedResource outside VelaQL right? If you can completely remove the existence of appliedResource, it would be ok to reuse the name. Actually, when I first saw this function, I was guessing this VelaQL provider is trying to load resources from .Status.appliedResources. It is a bit ambiguous. On the other hand, since this function is not exposed to user, the ambiguity only exists for KubeVela developers. With the comments above the function, it is ok to keep this name.